releasenotes/notes/ddtrace-1.0-disable-basic-config-call-by-default-b677844e4a13a794.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Default value of ``DD_CALL_BASIC_CONFIG`` was updated to `False`. Set ``DD_CALL_BASIC_CONFIG`` configuration to ``True`` to enable the pre-exisiting behavior.
```

Comment:
"to enable the pre-existing behavior" seems vague. Any thoughts on how I should reword this?